src/Combinatorics/Matroids/properties.jl

```diff
@@ -918,7 +918,7 @@ q^3 - 7*q^2 + 14*q - 8
 ```
 """
 function characteristic_polynomial(M::Matroid)
```

Comment:
These functions touched here should all at some point be changed allow passing in the polynomial ring, as discussed [here in our dev docs](https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Optional-arguments-for-parents-of-return-values)

(To be clear, that's not a request to change this PR)